ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/ActionManager.java

```diff
@@ -39,9 +39,28 @@ public void remove(Uid id)
 		_allActions.remove(id);
 	}
 
-    public int getNumberOfInflightTransactions()
+	/**
+	 * <p>
+	 *     This method works out the number of in-flight transactions currently in
+	 *     the system.
+	 * </p>
+	 * <p>
+	 *     Note: in-flight transactions are defined as all those transactions that
+	 *     haven't started the second phase of the two-phase commit (2PC) protocol.
```

Comment:
This implies that preparing and prepared transactions will be considered in flight.

Reply:
It is a rephrased version of "[active/in-flight is defined as] the state of a transaction when processing is in progress and completion of the transaction has not yet commenced." (from the OMG OTS specs). Do you think it should be rephrased?

Reply:
My bad. I'll change this javadoc